lsp-mode.el

```diff
@@ -8459,8 +8463,11 @@ archive (e.g. when the archive has multiple files)"
 
 (defcustom lsp-unzip-script (lambda ()
                               (cond ((and (eq system-type 'windows-nt)
-                                          (executable-find "powershell"))
+                                          (executable-find "pwsh"))
                                      lsp-ext-pwsh-script)
+                                    ((and (eq system-type 'windows-nt)
+                                          (executable-find "powershell"))
```

Comment:
this is for powershell 5 @jcs090218

Reply:
Thanks for adding this! I guess line 8472 is no longer needed? 🤔

Reply:
> Thanks for adding this! I guess line 8472 is no longer needed? 🤔

Sorry, I forgot to modify line 8472. Powershell 5 only supports windows. This line should use pwsh.

Reply:
I resquashed and the problem was fixed
The current logic is that: 
if you are on the windows platform, the priority of decompressing the zip is pwsh->powershell->unzip(bash)
other platform: unzip(bash)->pwsh